src/other/tables.jl

```diff
@@ -11,6 +11,10 @@ Tables.materializer(df::AbstractDataFrame) = DataFrame
 
 getvector(x::AbstractVector) = x
 getvector(x) = collect(x)
+fromcolumns(x::Tables.CopiedColumns; kw...) =
```

Comment:
Better not accept any keyword argument but ignore them: 
```suggestion
fromcolumns(x::Tables.CopiedColumns; copycols::Bool=true) =
```

Reply:
Why? It's an internal function and I was just trying to emphasize that `copycols` is ignored in this case.

Reply:
Well it's a matter of taste, but `kw...` isn't very explicit.

Reply:
kk, updated the keyword arg and added a comment that `copycols` is ignored for `CopiedColumns`